sr_world_generator/package.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>sr_world_generator</name>
+  <version>0.0.0</version>
+  <description>The sr_world_generator package</description>
+
+  <!-- One maintainer tag required, multiple allowed, one person per tag -->
+  <!-- Example:  -->
+  <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
+  <maintainer email="user@todo.todo">user</maintainer>
```

Comment:
Maybe something like ` <maintainer email="software@shadowrobot.com">>Shadow Robot's software team</maintainer> ` ?

Reply:
fixed in  fixed in 6565fc0